### PR TITLE
Add XpTracker toggle for tracking post-99 XP

### PR DESCRIPTION
**Pull request by evansloan**
Add a config option to opt out of tracking XP for stats that have
reached level 99.

[Preview](https://streamable.com/t5t7z)

### CODE REVIEW

File: runelite-client/src/main/java/net/runelite/client/plugins/xptracker/XpTrackerConfig.java [hunk 1]
```diff
@@ -52,4 +52,15 @@ default int pauseSkillAfter()
...
 		return 0;
 	}
+
+	@ConfigItem(
+		position = 2,
+		keyName = "trackSkillAfter99",
+		name = "Track skill after level 99",
+		description = "Configures whether or not skills will be tracked after level 99"
+	)
+	default boolean trackSkillAfter99()
```
Comment (deathbeam) [inline runelite-client/src/main/java/net/runelite/client/plugins/xptracker/XpTrackerConfig.java]: I would inverse this, e.g rename to Do not tracker after 99, and make it disabled by default

File: runelite-client/src/main/java/net/runelite/client/plugins/xptracker/XpTrackerConfig.java [hunk 2]
```diff
@@ -52,4 +52,15 @@ default int pauseSkillAfter()
...
 		return 0;
 	}
+
+	@ConfigItem(
+		position = 2,
+		keyName = "doNotTrackSkillAfter99",
+		name = "Do not track skill after level 99",
+		description = "Configures whether or not skills will be tracked after level 99"
+	)
+	default boolean doNotTrackSkillAfter99()
```
Comment (psikoi) [inline runelite-client/src/main/java/net/runelite/client/plugins/xptracker/XpTrackerConfig.java]: I think trackAfter99 would be simpler

Comment (raiyni) [reply runelite-client/src/main/java/net/runelite/client/plugins/xptracker/XpTrackerConfig.java]: I agree with this.

Comment (evansloan) [reply runelite-client/src/main/java/net/runelite/client/plugins/xptracker/XpTrackerConfig.java]: Yes, I'll change it in a bit.

**Comment by deathbeam** [comment]
Implemented in #5741

**Pull request closed without merging**